queries.sql

```diff
@@ -32,4 +32,74 @@ SELECT * FROM animals
 /* Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg) */
 SELECT * FROM animals
 	WHERE weight_kg >= 10.4
-		AND weight_kg <= 17.3;
\ No newline at end of file
+		AND weight_kg <= 17.3;
```

Comment:
* According to the [requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Inside%20a%20transaction%20update%20the%20animals%20table%20by%20setting%20the%20species%20column%20to%20unspecified.%20Verify%20that%20change%20was%20made.%20Then%20roll%20back%20the%20change%20and%20verify%20that%20the%20species%20columns%20went%20back%20to%20the%20state%20before%20the%20transaction.) you are to write queries that do the following inside your transactions, Kindly add them here 🙏 

> Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.

Reply:
Thanks for the information. I will update it